apps/chatbot/src/modules/utils.py

```diff
@@ -8,7 +8,7 @@
 
 AWS_ACCESS_KEY_ID = os.getenv("CHB_AWS_ACCESS_KEY_ID")
 AWS_SECRET_ACCESS_KEY = os.getenv("CHB_AWS_SECRET_ACCESS_KEY")
-AWS_DEFAUL_REGION = os.getenv("GOOGLE_AND_REDIS_AWS_DEFAULT_REGION")
+AWS_DEFAUL_REGION = os.getenv("CHB_GOOGLE_AWS_DEFAULT_REGION")
 
 
 def get_ssm_parameter(name: str, default: str | None = None) -> str | None:
```

Comment:
change `AWS_DEFAUL_REGION` with `AWS_DEFAULT_REGION` in row 26

Reply:
no

Reply:
I think @batdevis meant that there's a typo in the name: DEFAU**L**_REGION instead of DEFAU**LT**_REGION

Reply:
yes